src/viam/ur/module/ur_arm_state_disconnected.cpp

```diff
@@ -116,7 +146,7 @@ std::optional<URArm::state_::event_variant_> URArm::state_::state_disconnected_:
     }
 
     VIAM_SDK_LOG(info) << "disconnected: attempting recovery: recovery appears to have been successful; transitioning to independent mode";
-    return event_connection_established_{std::move(arm_connection)};
+    return arm_connection;
```

Comment:
so we don't need a move to pass the unique ptr? or does the `pending_connection.emplace` bit handle all of that?

Reply:
We don't need a move here because we are returning a unique_ptr, and returning like that automatically moves. The old code was returning an `event_connection_established_` and we needed to move the `arm_connection` into that object.